src/huggingface_hub/utils/_errors.py

```diff
@@ -15,6 +15,9 @@ class RepositoryNotFoundError(HTTPError):
     ```
     """
 
+    def __init__(self, message, response):
+        super(RepositoryNotFoundError, self).__init__(message, response=response)
```

Comment:
```suggestion
        super().__init__(message, response=response)
```
i think in Python 3 you just do this